UniversalAlgebra/Homomorphisms/Isomorphisms.lagda

```diff
@@ -48,10 +48,26 @@ private variable α β γ : Level
 
 Recall, `f ~ g` means f and g are *extensionally* (or pointwise) equal; i.e., `∀ x, f x ≡ g x`. We use this notion of equality of functions in the following definition of **isomorphism**.
 
-\begin{code}
+We could define this using Sigma types, like this.
 
+```agda
 _≅_ : {α β : Level}(𝑨 : Algebra α 𝑆)(𝑩 : Algebra β 𝑆) → Type(𝓞 ⊔ 𝓥 ⊔ α ⊔ β)
 𝑨 ≅ 𝑩 =  Σ[ f ∈ (hom 𝑨 𝑩)] Σ[ g ∈ hom 𝑩 𝑨 ] ((∣ f ∣ ∘ ∣ g ∣ ≈ ∣ 𝒾𝒹 𝑩 ∣) × (∣ g ∣ ∘ ∣ f ∣ ≈ ∣ 𝒾𝒹 𝑨 ∣))
+```
+
+However, with four components, an equivalent record type is easier to work with.
+
+\begin{code}
+
+record _≅_ (𝑨 : Algebra α 𝑆)(𝑩 : Algebra β 𝑆) : Type (𝓞 ⊔ 𝓥 ⊔ α ⊔ β) where
```

Comment:
the only extra thing to consider might be to give it a constructor, like say `mk≅` ? Then creating these becomes considerable shorter/less noisy than with explicit record syntax. It can be used for pattern matching too, which can sometimes be helpful.